artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -3404,6 +3404,14 @@
                   </xsd:documentation>
                </xsd:annotation>
             </xsd:element>
+
+            <xsd:element name="retroactive-message-count" type="xsd:long" default="0" maxOccurs="1" minOccurs="0">
```

Comment:
Can prefix default. In future or even in this pr it would be good to be able to explicitly set/control this at an address level. Thus this would just be a default thats then applied if none is set at that level